discussion.html

```diff
@@ -6,6 +6,8 @@
 <p>Please also make sure to check out
   the <a href='/faq.html'>FAQ</a>.</p>
 
+<p id="disqus_fallback">In case the comments do not work porperly, you can head over to the discussion at <a href="https://disqus.com/home/discussion/roca-style/roca_resource_oriented_client_architecture_discussion/">Disqus</a>.</p>
```

Comment:
This phrasing suggests graceful degradation rather than progressive enhancement; I'd just plunk the link there and remove it when (after?) loading Disqus.

Also, typo: `s/porperly/properly/`